app/components/homepage-components/Hero.tsx

```diff
@@ -27,8 +27,7 @@ const Hero = () => {
           setShowTick(false);
         }, 3000);
       })
-      .catch((err) => {
-        console.error("Could not copy text: ", err);
+      .catch(() => {
```

Comment:
Make sure you are still catching any errors and also informing the user... Looks good, though, just making sure.

Reply:
I now have an error toast here